index.html

```diff
@@ -0,0 +1,43 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+    <title>@Gopxfs Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+<body>
+  <div class ="page-flex">
+    <header class="header-flex">
+        <div class="header-logo-placeholder">
+            <p class="header-logo">John Doe</p>
```

Comment:
- [ ] Logo is almost there! Let's change `John Doe` for your actual name and add an `a` tag to make it clickable.